cognee/modules/retrieval/utils/description_to_codepart_search.py

```
@@ -62,7 +62,7 @@ async def code_description_to_code_part(
 
     try:
         if include_docs:
-            search_results = await search(query_text=query, query_type="INSIGHTS")
+            search_results = await search(query_text=query, query_type="GRAPH_COMPLETION")
```

Pylint (4.0.1)

[error] No value for argument 'dataset_ids' in function call

(E1120)